pkg/proxy/userspace/proxier.go

```diff
@@ -131,6 +131,8 @@ func createProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables
 	if proxyPorts == nil {
 		proxyPorts = newPortAllocator(util.PortRange{})
 	}
+	glog.V(2).Info("Tearing down pure-iptables proxy rules.")
```

Comment:
Self-Nit: I'm not sure if this should say "Errors are acceptable here." like we did in the iptables proxy.